frontend/ci-build-me/package.json

```diff
@@ -0,0 +1,53 @@
+{
+  "name": "github-webhook-cloud-function",
+  "version": "1.0.0",
+  "description": "Update Trello from GitHub Webhooks using Google Cloud Functions",
```

Comment:
small thing -- description is wrong
edit: actually much of the metadata in this package.json isn't updated for this project